x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -78,56 +98,87 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
     },
     {
       field: 'pValue',
-      name: 'p-value',
+      name: (
+        <EuiToolTip
+          position="top"
+          content={i18n.translate(
+            'xpack.aiops.correlations.failedTransactions.correlationsTable.pValueColumnTooltip',
+            {
+              defaultMessage:
+                'For statistically significant changes in the frequency of values, indicates how extreme the change is; lower values indicate greater change.',
```

Comment:
Do we show statistically insignificant changes in this list? If not, the tip can be shortened. For example:
```suggestion
                'The significance of changes in the frequency of values; lower values indicate greater change',
```